app/src/main/res/values/themes.xml

```diff
@@ -32,7 +32,7 @@
         </item>
     </style>
 
-    <style name="AppTheme.Dark" parent="Theme.MaterialComponents.NoActionBar">
+    <style name="AppTheme.Dark" parent="Theme.Material3.Dark.NoActionBar">
         <item name="colorPrimary">@color/white</item>
         <item name="colorSecondary">#2492d3</item>
         <item name="colorSecondaryVariant">#0065a1</item>
```

Comment:
Since we're only using compose for views, not layouts with android view, these colours shouldn't matter, the ones used in compose theme will be used anyway, I think maybe we should remove the color settings from themes.xml all to avoid confusion? 😉